dcp_prototype/backend/wrangling/migrations/scripts/migrate_dcp_one_to_dcp_two.py

```diff
@@ -48,37 +46,61 @@ def gather_group_file_list(file_list):
     In particular, links need to be processed last.
     This is done so that all entities exist before linkage and linking will not
     occur between unknown entities.
+
+    All files should end with .json except link files
     """
 
-    # group1 will contain all the non-links files, sorted by type, then by name
-    entity_group = []
-    # group2 contains just the links files.
-    link_group = []
+    entity_group = [fname for fname in file_list if fname.endswith(".json")]
+    link_group = [fname for fname in file_list if not fname.endswith(".json")]
+    return [entity_group, link_group]
 
-    tstart = time.time()
 
-    for file in file_list:
-        if "donor_organism" in file:
-            entity_group.append((1, file))
-        elif "specimen" in file:
-            entity_group.append((2, file))
-        elif "cell_suspension" in file:
-            entity_group.append((3, file))
-        elif "links" not in file:
-            entity_group.append((4, file))
-        elif "links" in file:
-            link_group.append(file)
-
-    entity_group.sort()
-    entity_group = [x[1] for x in entity_group]
-    link_group.sort()
+def process_file_data(filename, object_data, dataset_metadata):
+    tsv_generated_data_frame = json_normalize(flatten(object_data, separator="."))
+    entity_type = get_entity_type(filename, object_data)
+    if entity_type:
+        for row in tsv_generated_data_frame.iterrows():
+            metadata_row = row[1]
+            dataset_metadata.parse_flattened_row_of_json(metadata_row, entity_type)
 
-    tend = time.time()
-    print("group_file_list:", (tend - tstart))
-    return [entity_group, link_group]
 
+def consume_file(filequeue, dataset_metadata):
```

Comment:
Mind renaming to `consume_local_file`? Makes it clear in comparison to the other consume file functions.

Reply:
How about consume_file_local in keeping with the naming convention (consume_file_s3, consume_file_tar).